molecule/default/converge.yml

```diff
@@ -1,2 +1,6 @@
 ---
+- name: Include var file for import_playbook
+  hosts: all
+  tasks:
+    - ansible.builtin.include_vars: ../vars.yml
```

Comment:
Why not add `vars:` under `hosts:` (instead of using the `include_vars:`)?

Reply:
It also weird, because (to me), it means that there is a first playbook being executed called "Include var file.."

and then the imported playbook is executed, but I'm not sure the variables imported in the first one are still avalaible.

@guidograzioli Am I missing something here?

Reply:
Fixed.